content-repo/extra-docs/articles/Troubleshooting.md

```diff
@@ -0,0 +1,41 @@
+# Connection issues troubleshooting guide
+### When a customer has any connection errors try the following steps:
+1. Check and uncheck the “insecure” checkbox
+2. Check and uncheck the “proxy” checkbox
+3. Try to isolate the problem, is it the server or the client?
+
+    a. Try to generate the same request using curl or postman from your local computer (or customer’s computer)- if it fails: server is probably down, could be instance issue.
+    
+    b. Try to do some curl from the demisto machine, for example: 
+    ```
+   curl https://httpbin.org/status/200  -vk
```

Comment:
Why? The point of this step is to check if there is network at all.
Not to check if we can authenticate the integration instance.
IMHO there are too many ways to authenticate curl to specific integrations with basic authentication \ api key header so specifying it will not be enough to cover it so it's best not to start with it and keep it simple